mobile/lib/main.dart

```diff
@@ -136,8 +137,17 @@ class _TenTenOneAppState extends State<TenTenOneApp> {
     try {
       await setupRustLogging();
 
-      setState(() {
-        FLog.info(text: "10101 is ready!");
+      final appSupportDir = await getApplicationSupportDirectory();
+      FLog.info(text: "App data will be stored in: $appSupportDir");
+
+      api.run(appDir: appSupportDir.path).listen((event) {
```

Comment:
🙃 It might be a good idea to encapsulate these calls in a `service`.
For the `trade` feature I have an `order_service.dart` where the API calls are bundled; see https://github.com/get10101/10101/pull/143

Not sure it's worth it for these high-level calls, but it would make the dependencies more clear - otherwise it might be hard to manage these rust `api` calls over time, because they are sprinkled all over the code.